ros2lifecycle/package.xml

```diff
@@ -20,7 +20,10 @@
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>ament_xmllint</test_depend>
+
```

Comment:
Any particular reason for this newline?

Reply:
Not really, fixed in https://github.com/ros2/ros2cli/pull/391/commits/eb2f8d38f413ebd74fa7d91b1019e84487ba00ec